src/components/page-header.js

```diff
@@ -57,28 +58,17 @@ export default function PageHeader ({ breadcrumb, title, summary, image, ...rest
           </div>
           {imageData && (
             <React.Fragment>
-              <div
-                css={{
-                  backgroundImage: `url('${imageData.images.fallback.src}')`,
-                  backgroundPosition: 'center',
-                  backgroundRepeat: 'no-repeat',
-                  backgroundSize: 'cover',
-                  display: 'none',
-                  [MEDIA_QUERIES.LARGESCREEN]: {
-                    display: 'block'
-                  },
-                  flex: '0 1 50%'
-                }}
-              />
               <GatsbyImage
                 image={imageData}
                 css={{
+                  flex: '0 1 50%',
                   margin: `0 -${SPACING.M}`,
                   [MEDIA_QUERIES.LARGESCREEN]: {
-                    display: 'none'
+                    margin: 0,
+                    maxHeight: '350px'
```

Comment:
```suggestion
```
The `maxHeight` will not stretch the image to 100% height when viewing large screen, and the text begins to wrap (smaller than ~1285px). See Computer and Video Game Archive:
* [Preview](https://deploy-preview-308--future-wwwlib-previews.netlify.app/locations-and-hours/computer-and-video-game-archive)
* [Live](https://lib.umich.edu/locations-and-hours/computer-and-video-game-archive)

Reply:
Thanks, Erin. This was intentional to have close-to-parity with how this image is currently handled. It's currently a background-image and I'm changing it to an <img> so we can add alt text. The downside, as you've pointed out, is how it looks on the Computer and Video Game Archive page. I'll rework this so it's as close to prod as possible and report back soon...